src/WebJobs.Script/Scale/FunctionsScaleManager.cs

```diff
@@ -84,38 +92,36 @@ public virtual async Task<ScaleStatusResult> GetScaleStatusAsync(ScaleStatusCont
                 // this can happen if the host is offline
             }
 
-            var vote = GetAggregateScaleVote(votes, context, _logger);
+            var aggregatedVote = GetAggregateScaleVote(votes, context, _logger);
 
-            return new ScaleStatusResult
-            {
-                Vote = vote
-            };
+            return aggregatedVote;
         }
 
-        internal static ScaleVote GetAggregateScaleVote(List<ScaleVote> votes, ScaleStatusContext context, ILogger logger)
+        internal static int GetAggregateScaleVote(List<int> votes, ScaleStatusContext context, ILogger logger)
```

Comment:
I understand this pattern predates your PR, so address this comment at your discretion.

I am not a big fan of functions like this being marked `internal` to unit test them. It makes the class more rigid to refactors later on (as what should be completely opaque implementation details is now being verified by a test) and also allows others to take improper dependencies on this.

Do the unit tests for the public API surface of this class not sufficiently cover this code? If not, can they be adjusted to cover it? If covering this via public API surface is not possible, then maybe this helper is doing more than is necessary?